Modules/booking/Controller/BookingblockController.php

```diff
@@ -36,6 +36,19 @@ public function indexAction($id_space) {
         $modelColor = new BkColorCode();
         $colorCodes = $modelColor->getColorCodes($id_space);
 
+        $errorMessage = null;
+        if (empty($resources)) {
+            $errorMessage = BookingTranslator::ResourceNeeded($lang);
+        }
+        if (empty($colorCodes)) {
+            $errorMessage = $errorMessage ? $errorMessage . "</br>" : "";
+            $errorMessage .= BookingTranslator::ColorNeeded($lang);
```

Comment:
if resources but no color code will do .=  on null
if resource and colorcodes, errorMessage will be null, next message will not match against "" (null is != from "")

Reply:
- if resources but no color code, line 44 affects "" to $errorMessage
- ah yes, I forgot to add $erroMessage to the condition, like `if ($errorMessage && $errorMessage != "")`